roles/core/dns_client/templates/resolv.conf.j2

```diff
@@ -1,8 +1,16 @@
+#jinja2: lstrip_blocks: True
 #### Blue Banquise file ####
 ## {{ansible_managed}}
+{% set nameservers = networks[j2_node_main_network]['services_ip']['dns_ip'] | ipaddr %}
```

Comment:
I like the idea of filtering ip, to ensure no issues.

Reply:
See my previous comment: the ipaddr filter requires python's netaddr be installed on the ansible controller.
So if we want to keep this filter, we need to install the package on the ansible controller. Best option would be the ansible role.

Reply:
OK for netaddr. Useful.

Reply:
Done.

Reply:
Thanks ! 👍